src/plays/hangman-game/components/wordList.json

```diff
@@ -0,0 +1,51 @@
+[
```

Comment:
Good to not keep this file in `/components/` as it is not a react component.

Preferably use a `/constants/` and keep all data there.

Reply:
@sudipkundu999  done!